### DISCUSSION TITLE
为什么统计结果来看中西部省份偏多？

### DISCUSSION THREAD

**Discussion by freede953** [WeNeedHome/SummaryOfLoanSuspension#996] [Q&A]
恒大 融创 绿地 泰禾等开发商 都是全国性的开发楼盘，为什么统计的烂尾楼多的省份只是河南湖北湖南四川广西等中西部省份呢？

**Comment by ZVMAGL3** [comment]
因为东部房价还比较稳定，兜着不让降，开发商的现金流已经很危险了。

**Comment by jingzz1** [comment]
经济越落后的地区，政府越需要靠卖地增加收入，所以监管越宽松。
另外，开发商资金出了问题，肯定会优先保优质产业，郑州房价13k，深圳房价13w，在资金链出问题的情况下，你猜开发商会优先保哪个地方的楼盘？
